[Scanner] unexpected EOF Error in Version 6.5.0

It seems like there is an error in the templating of the scanner deployment. `helm lint` and `helm upgrade` show the following error:

```
[ERROR] templates/: parse error at (scanner/templates/scanner-deployment.yaml:144): unexpected EOF
```

This error results in failed deployments/upgrades.

--- a/scanner/templates/scanner-deployment.yaml
+++ b/scanner/templates/scanner-deployment.yaml
@@ -99,9 +99,10 @@ spec:
           value: "/opt/aquasec/ssl/key.pem"
         - name: "AQUA_PUBLIC_KEY"
           value: "/opt/aquasec/ssl/cert.pem"
-        {{- if .Values.cyberCenter.mtls.rootCA_fileName }}
+          {{- if .Values.cyberCenter.mtls.rootCA_fileName }}
         - name: "AQUA_ROOT_CA"
           value: "/opt/aquasec/ssl/ca.pem"
+          {{- end }}
         {{- end }}
         {{- include "scanner.extraEnvironmentVars" .Values | nindent 8 }}
         {{- include "scanner.extraSecretEnvironmentVars" .Values | nindent 8 }}

